deploy/pagerduty-service-orchestration/00-osd-serviceorchestration-configmap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+data:
+  service-orchestration.json: |
+    {
+    "orchestration_path": {
+    "catch_all": {
+        "actions": {}
+    },
+    "sets": [
+    {
+        "id": "start",
+        "rules": [
+        {     
+            "actions": {
+            "suppress": true
+            },
+            "conditions": [
+            {
+            "expression": "event.severity matches 'Warning'"
```

Comment:
I am not sure if it should be `warning` or `Warning`. From the alert payload. seems it should be `warning`